superset-frontend/src/features/databases/DatabaseModal/DatabaseConnectionForm/CommonParameters.tsx

```diff
@@ -116,6 +116,69 @@ export const databaseField = ({
     helpText={t('Copy the name of the database you are trying to connect to.')}
   />
 );
+export const defaultCatalogField = ({
+  required,
+  changeMethods,
+  getValidation,
+  validationErrors,
+  db,
+}: FieldPropTypes) => (
+  <ValidatedInput
+    id="default_catalog"
+    name="default_catalog"
+    required={required}
+    value={db?.parameters?.default_catalog}
+    validationMethods={{ onBlur: getValidation }}
+    errorMessage={validationErrors?.default_catalog}
+    placeholder={t('e.g. hive_metastore')}
+    label={t('Default Catalog')}
+    onChange={changeMethods.onParametersChange}
+    helpText={t('The default catalog that should be used for the connection.')}
+  />
+);
+export const defaultSchemaField = ({
+  required,
+  changeMethods,
+  getValidation,
+  validationErrors,
+  db,
+}: FieldPropTypes) => (
+  <ValidatedInput
+    id="default_schema"
+    name="default_schema"
+    required={required}
+    value={db?.parameters?.default_schema}
+    validationMethods={{ onBlur: getValidation }}
+    errorMessage={validationErrors?.default_schema}
+    placeholder={t('e.g. default')}
+    label={t('Default Schema')}
+    onChange={changeMethods.onParametersChange}
+    helpText={t('The default schema that should be used for the connection.')}
+  />
+);
+export const httpPathField = ({
```

Comment:
Had to create a new field for **HTTP Path** as the existing one included logic to retrieve/insert information in the `engine_params`